langfuse/client.py

```diff
@@ -179,8 +179,7 @@ def create_trace():
 
             def create_generation():
                 try:
-                    new_body = body.copy(update={"id": new_generation_id})
-                    new_body = body.copy(update={"trace_id": new_trace_id})
+                    new_body = body.copy(update={"id": new_generation_id, "trace_id": new_trace_id})
```

Comment:
Reason : 2 seperate generations were created from same trace because before this change when setting `trace_id` , `id` was set to None.

Reply:
same here!